js-api-doc/README.md

```diff
@@ -112,3 +120,38 @@ of Sass code by passing in a {@link LegacyStringOptions}.
     }
   });
   ```
+
+## Speed
+
+While multiple factors go into how long Sass compilations take, there are
+general speed trends that can help you minimize your compilation time.
+
+### With the `sass` package
+
+With the `sass` package, the synchronous calls will be faster than asynchronous
+calls due to the overhead of making the entire evaluation process asynchronous.
+While the {@link Compiler} and {@link AsyncCompiler} class are available, they
+aren't faster than than the module level compilation methods when using `sass`.
+
+### With the `sass-embedded` package
+
+The `sass-embedded` package provides significant speed improvements in certain
+situations, and is generally faster than `sass` for large compilations. When
+using the module level compilation methods, asynchronous calls are generally
+faster than synchronous ones due to the overhead of emulating synchronous
+messaging with worker threads and concurrent compilations being blocked on the
+main thread.
+
+The {@link Compiler} and {@link AsyncCompiler} classes provide significant
+improvements when using the `sass-embedded` package. We are able to persist and
+reuse a process across multiple compilations, avoiding the need to repeatedly
+start up and tear down a process.
```

Comment:
I'd frame this as the package doing the work rather than an ambiguous "we". Something like "These classes persist and reuse a single Dart process across multiple compilations"

Reply:
Fixed in c8b1198eeb33347f8c5df28e6a5ff798d5375c09